data/campaigns/Descent_Into_Darkness/scenarios/07_A_Small_Favor.cfg

```diff
@@ -208,20 +208,27 @@
             aggression=1.0
             caution=0.4
             grouping=offensive
-            [protect_area]
-                # Manor entrance
-                x,y=7,8
-                radius=5
+
+            [goal]
+                name=protect_location
+                [criteria]
+                    # Manor entrance
+                    x,y=7,8
+                [/criteria]
+                protect_radius=5
                 value=4
-            [/protect_area]
+            [/goal]
 
-            [protect_area]
-                # Rest of city
-                x=3-22
-                y=6-22
-                radius=1
+            [goal]
+                name=protect_location
+                [criteria]
+                    # Rest of city
+                    x=3-22
+                    y=6-22
+                [/criteria]
+                protect_radius=1
```

Comment:
@mattsc the two goals' areas overlap – is this fine because the one has a higher value?